packages/reports/addon/templates/components/report-view.hbs

```diff
@@ -1,18 +1,34 @@
-{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#if (feature-flag "enableRequestPreview")}}
+  <VisualizationSelector
+    @report={{@report}}
+    @validVisualizations={{validVisualizations}}
+    @onVisualizationTypeUpdate={{route-action "onVisualizationTypeUpdate"}}
+  />
+{{/if}}
 <div class="report-view__visualization-main">
   {{#if hasNoData}}
     <div class="report-view__visualization-no-results">
       No results available.
     </div>
   {{else}}
     <div class="report-view__visualization-header">
-      {{visualization-toggle
-        report=report
-        validVisualizations=validVisualizations
-        onVisualizationTypeUpdate=(route-action "onVisualizationTypeUpdate")
-      }}
+      {{#if (feature-flag "enableRequestPreview")}}
+        <span class="report-view__visualization-header__data-toggle">
+          <h3 class="report-view__visualization-header__title">
+            Data
+          </h3>
+          <NaviIcon @icon="angle-down" class="report-view__visualization-header__data-toggle-icon"/>
+        </span>
+      {{else}}
+        {{visualization-toggle
+          report=report
+          validVisualizations=validVisualizations
+          onVisualizationTypeUpdate=(route-action "onVisualizationTypeUpdate")
+        }}
```

Comment:
Angle brackets

Reply:
I didn't want to change any behavior with the `enableRequestPreview` flag disabled so I didn't want to change this. This component will be deprecated and deleted once the request preview is no longer feature flagged anyway.